server/api/controllers/question.js

```diff
@@ -0,0 +1,22 @@
+const Question = require("../models/question");
+
+exports.get = (req, res) => {
+  const limit = Number(req.query.limit);
+
+  if (!limit) {
+    res.status(404).json({
```

Comment:
404 doesn't make sense here
Let's use 400 which means Bad Request as this is not what we accept from the client.